README.md

```diff
@@ -6,7 +6,12 @@ Running latest all in one image
 docker run -d --name translate-all-in-one \
   -p 8080:8080 \
   -p 16686:16686 \
-  -e TRANSLATE_OTHER_GOOGLE_TRANSLATE_API_KEY={GoogleTranslate API key} \
+  -e TRANSLATE_OTHER_GOOGLE_TRANSLATE_PROJECT_ID={GoogleTranslate project id} \
+  -e TRANSLATE_OTHER_GOOGLE_TRANSLATE_LOCATION={GoogleTranslate location} \
+  -e TRANSLATE_OTHER_GOOGLE_TRANSLATE_ACCOUNT_KEY={GoogleTranslate account key} \
```

Comment:
This should go to Earthfile's `image-all-in-one` target with some default value e.g.
```dockerfile
ENV TRANSLATE_OTHER_GOOGLE_TRANSLATE_ACCOUNT_KEY=/app/google_account_key.json
```

And instead
```suggestion
  -v path/to/google_account_key.json:/app/google_account_key.json \
```

Reply:
Still stands